backend/main.py

```
@@ -26,122 +20,65 @@
-            # Start the bot as a background task.
-            asyncio.create_task(
-                self.discord_bot.start(settings.discord_bot_token)
-            )
-            logger.info("Background tasks started successfully!")
-        except Exception as e:
-            logger.error(f"Error during background task startup: {e}", exc_info=True)
-            await self.stop_background_tasks()
-            raise
+        asyncio.create_task(
+            self.discord_bot.start(settings.discord_bot_token)
+        )
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)